main.cpp

```diff
@@ -54,59 +54,59 @@ Thing: Car Wash
  */
 
 /*
-Thing 1)
+Thing 1) Crane
 5 properties:
-    1)
-    2)
-    3)
-    4)
-    5)
+    1) cabin for the operator to sit at the top of the machine
+    2) ladder to climb up to the cabin
+    3) horizontal arm to move the trolley
+    4) Counterweight to balance the arm 
+    5) Trolley to place the package in the z-axis 
```

Comment:
These are objects, not primitives!   You need to pick things that can be represented with `int float double bool char std::string`
- [ ] revise